README.md

```diff
@@ -80,24 +83,184 @@ After you input your application credentials, you can click the setup button, wh
 
 **Important**: Make sure your Resources/MobiledgeXSettings.asset file has the correct information for your application. 
 
-Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package : 
+
+### Example Usage:
+Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package 
+
+
+**Getting Edge Connection Url**
+
+MobiledgeX SDK uses the device Location and [the device MCC-MNC code](https://developers.mobiledgex.com/getting-started/connecting-client-app#distributed-matching-engine) to connect you to the closest Edge cloudlet were you application instance is deployed.
```

Comment:
minor: device => device's

I would also add MCC-MNC if avaliable

were you => where your